### PR TITLE
feat(mdx): gemoji

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix RM-9013 |
| :--------------------: | :---------: |

## 🧰 Changes

Adds MDX support for gemoji!

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: processor/transform/gemoji.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { emoji as Emoji } from '@readme/emojis';
+import { Root } from 'mdast';
+import { nameToEmoji } from 'gemoji';
```
Comment (kellyjosephprice) [inline processor/transform/gemoji.ts:3]: @Dashron with the new `moduleResolution: 'bunder'`, `tsc` is getting confused when looking up this module. It looks like it's resolving to the same file??

image

Comment (erunion) [reply processor/transform/gemoji.ts:3]: i've had issues with this in the past and haven't been able to find a way around it short of renaming the file i see this in to something else.

Comment (kellyjosephprice) [reply processor/transform/gemoji.ts:3]: Yea ok. I tried that but got confused cause I have other issues too.

**Review by trishaprile — COMMENTED** [review]
emojis look great! 
although owlmojis don't render for me
image

**Comment by kellyjosephprice** [comment]
> emojis look great! although owlmojis don't render for me image

I _think_ this is _okay_. Jenn is still working on images, so the owlmoji's won't work till then.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.28</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.28)
<br>

**Pull request merged**